test/modeles/utilisateur.spec.js

```diff
@@ -12,17 +12,52 @@ const { unUtilisateur } = require('../constructeurs/constructeurUtilisateur');
 
 describe('Un utilisateur', () => {
   describe("sur demande d'un profil complet ou non", () => {
-    it('considère le profil « complet » dès lors que le nom est renseigné', () => {
+    it("considère le profil « complet » dès lors que le nom et le siret de l'entite sont renseignés", () => {
       const utilisateur = new Utilisateur({
         nom: 'Dupont',
         email: 'jean.dupont@mail.fr',
+        entite: { siret: '12345' },
       });
-      expect(utilisateur.profilEstComplet()).to.be(true);
+      expect(utilisateur.completudeProfil().estComplet).to.be(true);
     });
 
     it("considère le profil « incomplet » si le nom n'est pas renseigné", () => {
```

Comment:
`it('considère le profil incomplet *à cause du nom* si…`

Et comme ça on rajoute le nouvel `expect` dans ce test.